apps/studio/src/features/dashboard/components/ResourceTable/ResourceTableMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { MenuButton, MenuList } from "@chakra-ui/react"
+import { IconButton, Menu } from "@opengovsg/design-system-react"
+import {
+  BiCog,
+  BiDotsHorizontalRounded,
+  BiDuplicate,
+  BiFolderOpen,
+  BiTrash,
+} from "react-icons/bi"
+
+import type { ResourceTableData } from "./types"
+import { MenuItem } from "~/components/Menu"
+
+interface ResourceTableMenuProps {
+  title: ResourceTableData["name"]
+  resourceId: ResourceTableData["id"]
+  type: ResourceTableData["type"]
+}
+
+export const ResourceTableMenu = ({ title, type }: ResourceTableMenuProps) => {
+  return (
+    <Menu isLazy size="sm">
```

Comment:
how come `Menu` can `isLazy`

Reply:
https://v2.chakra-ui.com/docs/components/menu/usage#lazily-mounting-menuitem

suggest reading docs first before raising comments hahaha if not looks like need to spoonfeed :P